tools/testbin

```diff
@@ -135,7 +135,7 @@ deb_start_test() {
     for PG_VERSION in $PG_VERSIONS; do
         select_pg $PG_VERSION
         deb=timescaledb-toolkit-postgresql-${PG_VERSION}=${EPOCH}${FROM_VERSION}~${OS_NAME}${OS_VERSION}
-        $nop sudo apt-get -qq install $deb
+        [ $1 = deb ] && $nop sudo apt-get -qq install $deb
```

Comment:
apt-get install is idempotent so there should be no reason to special-case this; did you hit a problem?

Reply:
In [this run](https://github.com/timescale/timescaledb-toolkit/actions/runs/3192423093/jobs/5209880700) (before I changed that line) I got an error:
```
+ sudo apt-get -qq install timescaledb-toolkit-postgresql-12=1.5.1~debian10
E: Version '1.5.1~debian10' for 'timescaledb-toolkit-postgresql-12' was not found
```

Reply:
That's because a few days ago when I wrote "we actually run on Debian 10!" I had mixed up bullseye and buster for the 50th time :)

We're actually on Debian 11.